### PR TITLE
[QSLPrint] Added export button for selected QSOs

### PR DESCRIPTION
**Pull request by AndreasK79**
image
Added another button for exporting selected QSOs.
Changed wording from printed to sent on another button.

### CODE REVIEW

File: application/views/qslprint/qslprint.php [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<script type="text/javascript">
+    const user_callsign = "<?php echo $this->session->userdata('user_callsign') ?>";
+</script>
```
Comment (HB9HIL) [inline application/views/qslprint/qslprint.php:3]: Let us move this to the footer.php so we can reuse this :)

**Review by HB9HIL — CHANGES_REQUESTED** [review]

File: application/views/qslprint/qslprint.php (hunk 1 above)
Comment (HB9HIL) [reply application/views/qslprint/qslprint.php:3]: I just noticed we already have a variable for this in the footer :)

```suggestion
```

File: assets/js/sections/qslprint.js [hunk 2]
```diff
@@ -244,3 +244,56 @@ function removeSelectedQsos() {
...
+
+			// Format the date and time as UTC Ymd-Hi
+			const year = now.getUTCFullYear();
+			const month = String(now.getUTCMonth() + 1).padStart(2, '0'); // Months are zero-based
+			const day = String(now.getUTCDate()).padStart(2, '0');
+			const hours = String(now.getUTCHours()).padStart(2, '0');
+			const minutes = String(now.getUTCMinutes()).padStart(2, '0');
+
+			// Create the formatted filename
+			const filename = `${user_callsign}-${year}${month}${day}-${hours}${minutes}.adi`;
```
Comment (HB9HIL) [inline assets/js/sections/qslprint.js:283]:
```suggestion
			const filename = `${my_call}-${year}${month}${day}-${hours}${minutes}.adi`;
```

**Review by HB9HIL — CHANGES_REQUESTED** [review]
the user_callsign from the session is already global available as `my_call` 
See footer.php Line 25

**Comment by AndreasK79** [comment]
> the user_callsign from the session is already global available as `my_call` See footer.php Line 25

Thanks. I didn't look closely enough :)

**Review by HB9HIL — APPROVED** [review]

**Comment by m0urs** [comment]
Thank you very much!

**Pull request merged**